src/platform/julius.c

```diff
@@ -509,20 +513,12 @@ static void teardown(void)
 
 int main(int argc, char **argv)
 {
-    const char *custom_data_dir = NULL;
-    for (int i = 1; i < argc; i++) {
-        // we ignore "-psn" arguments, this is needed to launch the app
-        // from the Finder on macOS.
-        // https://hg.libsdl.org/SDL/file/c005c49beaa9/test/testdropfile.c#l47
-        if (SDL_strncmp(argv[i], "-psn", 4) == 0) {
-            continue;
-        }
-
-        custom_data_dir = argv[i];
-        break;
+    julius_args args;
+    if (!platform_parse_arguments(argc, argv, &args)) {
+        return 1;
```

Comment:
Should julius really exit on invalid cmd arguments? How about logging a warning and proceeding to use default values?

Reply:
I was doubting this one. Command-line programs usually terminate when you call them with invalid arguments. On the other hand: Julius is not a command-line program and expected to launch from UI as well.

Reply:
I decided not to exit on invalid args and continue with the defaults.